Add dynatrace integration

jm-dt opened the pull request:
Adds dependency on libbuildpack-dynatrace
Adds dynatrace hook
Adds integration tests
Adds test fixures

jm-dt committed:
Add dynatrace integration

meibensteiner commented:
This PR has been untouched for 2 months. Based on recent commits: 
@arjun024 @cf-buildpacks-eng @ForestEckhardt Could you help out here?

arthfl commented:
We're still waiting on at least *some* reaction here :-/

jm-dt committed:
Merge branch 'cloudfoundry:master' into master

jm-dt committed:
Update to libbuildpack-dynatrace 1.8.1

jm-dt committed:
Update libbuildpack-dynatrace to v1.9.0

tnikolova82 committed:
Fix vendoring: add missing unix.go and windows.go from libbuildpack-dynatrace

The vendor directory was incomplete - missing platform-specific files
that define runInstallerUnix and runInstallerWindows methods, causing
compilation failures on all platforms.
